pkg/cmd/experimental/bundlesecret/bundle-secret.go

```diff
@@ -0,0 +1,114 @@
+package bundlesecret
+
+//export OPENSHIFTCONFIG=./openshift.local.certificates/admin/.kubeconfig
+import (
+	"fmt"
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	kcmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	"github.com/spf13/cobra"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+const MaxSecretSize = 1 * 1024 * 1024
```

Comment:
I mean a 1GB file base64 encoded is never a problem ;-)  Either way, I still think for simplicity its reasonable to get the basic function in the command settled, and then doing things like ignore files with invalid names, or ignore files that are larger than X can all be handled after based on some initial experience.  In the end, this command could have a man page the size of tar ;-)